icons/play.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import type { Variants } from 'framer-motion';
+import { motion, useAnimation } from 'framer-motion';
+
+const pathVariants: Variants = {
+  normal: {
+    opacity: 1,
+    x: 0,
+    rotate: 0,
+  },
+  animate: {
+    x: [0, -1, 2, 0],
+    rotate: [0, -10, 0, 0],
+    opacity: 0.5,
```

Comment:
why did you decide to change the opacity? Ii don't think we should change it, it's all up to the user

Reply:
@pqoqubbw Oh, I’m sorry. I didn’t mean to do that either, and it was something I briefly applied during testing, but it ended up being included in the PR. I’ll make the necessary updates!